scraper/00cef875-17bf-4ca1-985e-571fe3bd65e9.yaml

```diff
@@ -0,0 +1,29 @@
+- - Where can I submit my application materials electronically?
+  - Please refer to the<a href="/undergrad/apply/freshmen-instructions/">First-Year Application Instructions</a>or the<a href="/undergrad/apply/transfer-instructions/">Transfer Application Instructions</a>for detailed instructions on how to submit individual items.
+- - Where should I mail my application materials?
+  - Please refer to the<a href="/undergrad/apply/freshmen-instructions/">First-Year Application Instructions</a>or the<a href="/undergrad/apply/transfer-instructions/">Transfer Application Instructions</a>for our mailing addresses.
+- - How do I apply for scholarships?
+  - International students are automatically considered for merit aid during the application review process. No additional paperwork is required. Merit scholarship decisions are based on the credentials submitted at the time of application. Students who are awarded a merit scholarship will be notified in their acceptance letter. If you are interested in need-based aid, you must complete the<a href="http-//cssprofile.collegeboard.org/">CSS Profile</a>by the appropriate deadline. To view the deadlines and find more information regarding the CSS Profile, please visit the<a href="/undergrad/financing/apply/">How to Apply for Aid</a>page.
+- - I cannot cover the entire cost of a Drexel education. Can I have a job on campus?
+  - International students on an F1 visa may work on campus. However, we cannot guarantee that you will be able to find a job. Unfortunately, we are unable to provide a job assignment to you and it will be your responsibility to seek out and apply for on-campus jobs if you wish to have one.
+- - I am an international student and will have an F1 visa. Can I do co-op?
+  - Yes, you are eligible to participate in the co-op program if you have an F1 visa.
+- - Will participating in the co-op program affect my Optional Practical Training (OPT)?
+  - If you are enrolled in the One Co-op (Four Years) option, it will not affect your OPT. If you are enrolled in the Three Co-op (Five Years) option, it will affect your OPT if you do all of your co-ops full-time in the United States. In order to preserve their OPT for after graduation, students with three co-ops will often do one of their co-ops part-time or one of them abroad. For more information regarding your OPT, please contact our International Students and Scholars Services (ISSS) office at<a href="mailto-isss@drexel.edu">isss@drexel.edu</a>or by phone at 215.895.2502.
+- - I am not on a student visa. I have an H4 (L2, etc.) visa. How does co-op work for me? Can I participate?
+  - Please contact the ISSS office regarding this issue. They can be reached by email at<a href="mailto-isss@drexel.edu">isss@drexel.edu</a>or by phone at 215.895.2502.
```

Comment:
There is html markup here. at<a href="mailto-isss@drexel.edu">. This might be a problem for out ChatBot engine. Further investigation might be required to see if this would cause any issues.